lib/health_web/templates/layout/app.html.eex

```diff
@@ -18,7 +18,7 @@
       <div class="container">
         <%= render "_alert.html", conn: @conn %>
 
-        <div id="main">
+        <div id="main" class="cont">
```

Comment:
every page you have starts with `div class="cunt"` and it should probably just be in the layout?

Reply:
Hah! ... probably.
I was thinking of making the the big graph page full screen but am now doubting myself.

I'll put it in the issues.